test/system/test_tpm2_nv.sh

```diff
@@ -85,3 +86,17 @@ if [ $? != 0 ];then
 else
  echo "release the nv index OK!"
 fi
+
+echo "f28230c080bbe417141199e36d18978228d8948fc10a6a24921b9eba6bb1d988" \
+| xxd -r -p > policy.bin
+tpm2_nvdefine -x 0x1500016 -a 0x40000001 -s 32 -t 0x2000A -L policy.bin -r -w
+tpm2_nvlist | grep 0x1500016 -A5 | grep Auth | grep -o ": [a-zA-Z0-9]\{1,\}" | \
+grep -o "[a-zA-Z0-9]\{1\}" | xxd -r -p >test.bin
+tpm2_nvrelease -x 0x1500016 -a 0x40000001
```

Comment:
Should we be checking the return codes above? Should we let it fail all the way to here?